setup.py

```diff
@@ -26,8 +26,8 @@
     url='https://github.com/boykoc/ckanext-ontario_theme',
```

Comment:
this should be updated to `https://github.com/ongov/ckanext-ontario_theme`

Reply:
sorry! missed that